Gemfile

```diff
@@ -101,8 +101,13 @@ gem "flipper-active_record"
 gem "flipper-ui"
 # Calculates latitude and longitude from an address.
 gem "geocoder"
+# Enable making HTTP requests
+gem 'httparty'
 # JSON Web Token encoding / decoding (e.g. for links in e-mails)
 gem "jwt"
+# A HTML scraping DSL to help with extraction NDBN member ids
+# from their website
+gem 'scraping'
```

Comment:
I see why this was added, but I'm wondering if we can make do with a simple Nokogiri xpath query (we already have nokogiri in the app). I see a trend in this app of adding gems that are used once or twice and don't add a ton of value in their usage. If we thought we'd be doing more web scraping, it'd be a different story, but I don't see anything in the roadmap indicating this.

Reply:
Very good point! I forgot that Nokogiri was an option from the start. I agree that there is a trend to adding gems without needing it.